test/functional/wallet_basic.py

```diff
@@ -653,6 +653,16 @@ def run_test(self):
         assert_array_result(tx["details"], {"category": "receive"}, expected_receive_vout)
         assert_equal(tx[verbose_field], self.nodes[0].decoderawtransaction(tx["hex"]))
 
+        self.log.info("Testing Fee Reason")
```

Comment:
nit: log message can be more informative, like "Test send* RPCs with verbose=True"

Reply:
Also then comments are not _that_ necessary since it's pretty self-explanatory what you're doing

Reply:
Style-wise, if you keep comments, it's best for them to be complete sentences